packages/breadcrumbs/Breadcrumbs.js

```diff
@@ -21,23 +22,26 @@ class Breadcrumbs extends React.Component {
     );
   };
 
-  render() {
-    const { crumbs, active, emptyState } = this.props;
-
-    return (
-      <Breadcrumb>
-        <BreadcrumbItem>
-          <a aria-label="Home" href="/public/apps/dashboard">
-            Home
-          </a>
-        </BreadcrumbItem>
-        {crumbs && crumbs.length > 0 && crumbs.map(this.renderBreadCrumb)}
-        {this.props.children}
-        <BreadcrumbItem active>{active || emptyState}</BreadcrumbItem>
-      </Breadcrumb>
-    );
-  }
-}
+  return (
+    <Breadcrumb>
+      <BreadcrumbItem>
+        <a aria-label="Home" href="/public/apps/dashboard">
+          Home
+        </a>
+      </BreadcrumbItem>
+      {crumbs && crumbs.length > 0 && crumbs.map(renderBreadCrumb)}
+      {children}
+      <BreadcrumbItem active>{active || emptyState}</BreadcrumbItem>
+      {feedback && (
+        <Feedback
+          appName={feedback.appName || active}
```

Comment:
Is 'active' the right backup here? Im assuming 'active' is the name of the active page, not necessarily the 'appName'.

Reply:
Right, it can go both ways but the idea was that you should be submitting feedback generally about the app and not the "page". So if you had a feedback component on the Home page it doesn't make sense to display "`Tell us what you think about Home`. Although this can also be overriden in the props for the header text of the feedback.

Reply:
To keep the data consistent, appName should always be the same.

Reply:
Agreed with @sanelo. The appName will be sent in the Splunk analytics and we should just require the appName on the feedback props to ensure the right analytics get logged.